aion/web/elm/src/Panel/View.elm

```diff
@@ -116,3 +126,26 @@ categoryNameFormElement form =
             ]
         , Badge.pillInfo [] [ text (Forms.errorString form "name") ]
         ]
+
+
+
+-- room form section
+
+
+roomFormElement : Forms.Form -> Html Msg
+roomFormElement form =
+    Form.group []
+        [ Form.label [ for "room" ] [ text "Enter room name bellow, should be uppercase:" ]
```

Comment:
I made a special issue for correcting things like `bellow`, but it can be fixed before merging anyway #110